packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.html

```diff
@@ -0,0 +1,43 @@
+<div data-ui-view>
+    <oui-header
+        class="mb-3"
+        data-heading="{{:: 'pci_projects_project_storages_cold_archive_title' | translate }}"
+        data-description="{{:: 'pci_projects_project_storages_cold_archive_description' | translate }}"
+    >
+        <oui-guide-menu
+            data-text="{{:: 'pci_project_guides_header' | translate }}"
+        >
+            <oui-guide-menu-item
+                data-ng-repeat="guide in $ctrl.guideUrl.storage track by $index"
```

Comment:
Use of `$ctrl` but I don't see any controller imported or bindings named `guideUrl`

Reply:
Right, missed during copy-past.